### PR TITLE
Update to node 10

### PR DESCRIPTION
**Pull request by timja-kainos**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -79,7 +77,7 @@
 
 "@sinonjs/formatio@^2.0.0":
   version "2.0.0"
-  resolved "https://registry.yarnpkg.com/@sinonjs/formatio/-/formatio-2.0.0.tgz#84db7e9eb5531df18a8c5e0bfb6e449e55e654b2"
+  resolved "http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz#84db7e9eb5531df18a8c5e0bfb6e449e55e654b2"
```
Comment (geek-soft) [inline yarn.lock]: That is interesting. I wonder why yarn change domain.

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "1.1.1",
   "private": true,
   "engines": {
-    "node": ">=8.9.4 <10.0.0"
+    "node": ">=8.9.4"
```
Comment (sabahirfan) [inline package.json]: should this be ` >=10.1.0`?

Comment (timja-kainos) [reply package.json]: nah need jenkins to be upgraded for that, which requires all the teams to update

Comment (sabahirfan) [reply package.json]: ok

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM node:8.9.4-alpine
+FROM node:10.1.0-alpine
```
Comment (sabahirfan) [inline Dockerfile]: The same need to be updated in `integration-tests.Dockerfile` as well

**Comment by timja-kainos** [comment]
deferred for now

**Pull request closed without merging**